eZ/Publish/SPI/Search/Handler.php

```diff
@@ -11,6 +11,8 @@
 
 /**
  * The interface for the main Search Engine handlers
+ *
+ * @todo workshop: merge content and location search handler
```

Comment:
Merged in https://github.com/ezsystems/ezpublish-kernel/commit/4bbd51aaf8f05f7a8e767b3e54f046a1fac7fdfe and https://github.com/ezsystems/ezplatform-solr-search-engine/commit/3356aeec9af7ff19bc01915058158646e336e403